src/App.svelte

```diff
@@ -2,6 +2,14 @@
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
   console.log(exampleData);
+  export function grabip(text) {
+    let textbox = document.createElement('textarea');
+    textbox.value= text['ip'];
+    document.body.append(textbox);
+    textbox.select();
+    document.execCommand("copy");
+    textbox.remove();
+  } 
```

Comment:
Why are you passing it as an object and then indexing into the object?

Reply:
Also can you verify this works across browsers/recent versions? If it doesn't we could just use navigator+polyfill ```await navigator.clipboard.writeText(ip);```

Reply:
Use modern api's + polyfills when possible

Reply:
Also, design thing - we might want a visual indicator that its been copied. Better for discoverability

Reply:
pretty sure this code has greater compatibility across browsers than writeText

Reply:
@ming08108 said he wanted to get rid of libraries